src/allmydata/scripts/common.py

```diff
@@ -270,6 +273,20 @@ def get_alias(aliases, path_unicode, default):
     return uri.from_string_dirnode(aliases[alias]).to_string(), path[colon+1:]
 
 def escape_path(path):
-    # this always returns bytes, specifically US-ASCII, valid URL characters
+    # type: (str) -> str
+    u"""
+    Return path quoted to US-ASCII, valid URL characters.
+
+    >>> path = u'/føö/bar/☃'
+    >>> escaped = escape_path(path)
+    >>> str(escaped)
+    '/f%C3%B8%C3%B6/bar/%E2%98%83'
+    >>> escaped.encode('ascii').decode('ascii') == escaped
+    True
+    """
     segments = path.split("/")
-    return "/".join([urllib.quote(unicode_to_url(s)) for s in segments])
+    result = "/".join([urllib.parse.quote(unicode_to_url(s)) for s in segments])
+    # fixme: test.cli.test_create_alias fails if it gets Unicode on Python 2
+    if PY2 and isinstance(result, type(u'')):
+        result = result.encode('ascii')
```

Comment:
These two lines could be `six.ensure_str(result, "ascii")` right?

Also I'm not sure what the fixme is about.  is the test broken?  is this code still broken?  is this just about needing to finish porting all this code to python 3?  I suggest expanding the comment to explain what action can be taken to fix the issue, expanding the comment into a complete ticket, or deleting it if there's no particular action to take apart from finishing the port.